lib/matplotlib/axes/_axes.py

```diff
@@ -4104,112 +4095,132 @@ def hexbin(self, x, y, C=None, gridsize=100, bins=None,
-           :class:`~matplotlib.collections.RegularPolyCollection`.
+        mincnt : int > 0, optional, default is *None*
+            If not *None*, only display cells with more than *mincnt*
+            number of points in the cell
 
-        Other keyword arguments controlling the Collection properties:
+        marginals : bool, optional, default is *False*
+            if marginals is *True*, plot the marginal density as
+            colormapped rectagles along the bottom of the x-axis and
+            left of the y-axis
 
-        *edgecolors*: [ *None* | ``'none'`` | mpl color | color sequence ]
-           If ``'none'``, draws the edges in the same color as the fill color.
-           This is the default, as it avoids unsightly unpainted pixels
-           between the hexagons.
+        extent : scalar, optional, default is *None*
+            The limits of the bins. The default assigns the limits
+            based on *gridsize*, *x*, *y*, *xscale* and *yscale*.
 
-           If *None*, draws the outlines in the default color.
+            If *xscale* or *yscale* is set to 'log', the limits are
+            expected to be the exponent for a power of 10. E.g. for
+            x-limits of 1 and 50 in 'linear' scale and y-limits
+            of 10 and 1000 in 'log' scale, enter (1, 50, 1, 3).
 
-           If a matplotlib color arg or sequence of rgba tuples, draws the
-           outlines in the specified color.
+            Order of scalars is (left, right, bottom, top).
 
-        Here are the standard descriptions of all the
-        :class:`~matplotlib.collections.Collection` kwargs:
+        Other parameters
+        ----------------
+        cmap : object, optional, default is *None*
+            a :class:`matplotlib.colors.Colormap` instance. If *None*,
+            defaults to rc ``image.cmap``.
+
+        norm : object, optional, default is *None*
+            :class:`matplotlib.colors.Normalize` instance is used to
+            scale luminance data to 0,1.
+
+        vmin : scalar, optional, default is *None*
+            *vmin* is used in conjunction with *norm* to normalize
+            luminance data. If *None*, the min of the color array *C*
+            is used.  Note if you pass a norm instance, your settings
+            for *vmin* will be ignored.
+
+        vmax : scalar, optional, default is *None*
+            *vmax* is used in conjunction with *norm* to normalize
+            luminance data. If *None*, the max of the color array *C*
+            is used.  Note if you pass a norm instance, your settings
+            for *vmax* will be ignored.
+
+        alpha : scalar between 0 and 1, optional, default is *None*
+            the alpha value for the patches
+
+        linewidths : scalar, optional, default is *None*
+            If *None*, defaults to 1.0. Note that this is a tuple, and
+            if you set the linewidths argument you must set it as a
+            sequence of floats, as required by
```

Comment:
This is repeating the original incorrect description.  In fact, only a scalar makes sense here, although a sequence can be accepted if it is not too long.  I can't see any reasonable use case for a sequence of length other than 1, for which one might as well use a scalar.  So, please delete the sentence starting with "Note".